pilot/cmd/pilot-discovery/main.go

```diff
@@ -120,6 +120,8 @@ func init() {
 	serverArgs.MCPCredentialOptions.AttachCobraFlags(discoveryCmd)
 	discoveryCmd.PersistentFlags().IntVar(&serverArgs.MCPMaxMessageSize, "mcpMaxMsgSize", bootstrap.DefaultMCPMaxMsgSize,
 		"Max message size received by MCP's grpc client")
+	discoveryCmd.PersistentFlags().BoolVar(&serverArgs.IncrementalEDS, "incrementalEDS", false,
+		"Enable incremental EDS updates to Envoy")
```

Comment:
Well - 1.1 only incremental EDS to envoy is supported, this would be quite missleading.

Reply:
That flag is going away.